src/doc/rustc/src/command-line-arguments.md

```diff
@@ -62,11 +62,15 @@ attribute or on the command-line, it will link a dynamic library if available,
 otherwise it will use a static library. If the kind is specified on the
 command-line, it will override the kind specified in a `link` attribute.
 
+Note that if you don't specify the `KIND`, the default linkage modifiers won't be applied.
```

Comment:
What does "default linkage modifiers" mean?
There are no default linkage modifiers.

Reply:
I meant this. It's enabled by default for `static` kind https://github.com/rust-lang/rust/blob/46c5ee11bb47de0cba0b4538537e1e3813088583/src/doc/rustc/src/command-line-arguments.md?plain=1#L109

Reply:
```suggestion
If you don't specify `KIND`, no linkage modifiers will be applied.
```

This might be a slightly clearer wording..? If I understand correctly, when `KIND` is not specified then the `+` or `-` default for the modifiers doesn't apply.